.github/workflows/lint.yml

```
@@ -0,0 +1,26 @@
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+
+      - name: Install Ruff
+        run: pip install ruff
+
+      - name: Run Ruff Lint with Fix
+        run: ruff check . --fix
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)